src/heroku/build_docker_image.ts

```diff
@@ -6,20 +6,21 @@ export const buildDockerImage = async ({
   dockerOptions,
   herokuAppName,
   cwd,
-  processType,
+  processTypes,
 }: {
   dockerfileName: string;
   dockerOptions: string;
   herokuAppName: string;
   cwd: string;
-  processType: string;
+  processTypes: string[];
 }): Promise<boolean> => {
   try {
     core.startGroup('Building docker container...');
 
+    const tags = processTypes.map(processType => `--tag registry.heroku.com/${herokuAppName}/${processType}`);
+
     await runCommand(
-      `docker build --file ${dockerfileName} ${dockerOptions} ` +
-        `--tag registry.heroku.com/${herokuAppName}/${processType} .`,
+      `docker build --file ${dockerfileName} ${dockerOptions} ${tags.join(' ')} .`,
       { options: { cwd } },
```

Comment:
Hi @danthedaniel,
I've got one question for you. Do we want to tag the same build with different tags? I mean... that this will cause our `release` and `web` processes to be exactly the same images 🤔

Reply:
That's how I use this, yes.

My entrypoint script has to account for that:

```shell
#!/bin/bash

cd /opt

# Release process
if [[ "$DYNO" =~ ^release.* ]]; then
  # ...

  exit 0
fi

# Web process
# ...
```